tests/unit_tests/actsvg/actsvg_masks.cpp

```diff
@@ -0,0 +1,95 @@
+
+#include <array>
+#include <fstream>
+#include <iostream>
+#include <string>
+#include <type_traits>
+#include <vector>
+
+// Project include(s)
+#include "detray/core/detector.hpp"
+#include "detray/definitions/indexing.hpp"
+#include "detray/definitions/qualifiers.hpp"
+#include "detray/detectors/create_toy_geometry.hpp"
+#include "detray/geometry/surface.hpp"
+#include "detray/io/common/detector_writer.hpp"
+#include "detray/masks/cylinder2D.hpp"
+#include "detray/masks/masks.hpp"
+#include "detray/plugins/actsvg/surface_svg_converter.hpp"
+#include "detray/tracks/tracks.hpp"
+
+// Vecmem include(s)
+#include <vecmem/memory/host_memory_resource.hpp>
+
+// Actsvg include(s)
+#include "actsvg/core.hpp"
+#include "actsvg/core/defs.hpp"
+#include "actsvg/display/geometry.hpp"
+#include "actsvg/meta.hpp"
+#include "actsvg/proto/surface.hpp"
+#include "actsvg/styles/defaults.hpp"
+
+using namespace actsvg;
+
+using point3 = std::array<scalar, 3>;
```

Comment:
Changing the type in future maintainance will become easier if it defined in just one place and then handed through:
```suggestion
using point3 = detray::actsvg_visualization::point3;
```